pauthabielf64/pauthabielf64.rst

```diff
@@ -1092,6 +1092,18 @@ Example Code to access a signed GOT entry
 In the example the :got_auth: and :got_auth_lo12: operators result in
 AUTH variant GOT generating relocations being used.
 
+Example Code to access a signed GOT entry with the tiny code model:
+
+.. code-block:: asm
+
+  adr x8, :got_auth: symbol
+  ldr x0, [x8]
+  // Authenticate to get unsigned pointer
+  autia x0, x8
+
+Compared to the tiny code model without pointer authentication an
+additonal adr is required to get the address of the GOT entry.
```

Comment:
I'm not sure I am following here - all PLT sequences as currently defined(1) must compute the address of the GOT entry for the dynamic linker. So the ADR is required even without PAUTH, and it would be worth having the equivalent relocation if it doesn't already exist (given that just having PC-relative LDR won't work).

(1) It is possible to define PLT sequences that compute the address of the GOT entry in a different way.

Reply:
You're right, there doesn't seem a case for user code, so if linkers use internal relocations then there is no need to add one.